lua/cmp/view/native_entries_view.lua

```diff
@@ -123,6 +123,7 @@ native_entries_view.select_next_item = function(self, option)
     else
```

Comment:
We should remove `CompleteChanged` in the above of lines (maybe)

Reply:
Are you talking about:

```rust
  autocmd.subscribe('CompleteChanged', function()
    self.event:emit('change')
  end)
```

I think we would still need this for when the actual content of the popup menu has changed, no?